jx-docker-build.sh

```diff
@@ -98,12 +98,16 @@ if [ "release" == "${RELEASE}" ]; then
     jenkinsxio/builder-go ${VERSION} \
     jenkinsxio/builder-go-maven ${VERSION} \
     jenkinsxio/builder-gradle ${VERSION} \
+    jenkinsxio/builder-gradle4 ${VERSION} \
```

Comment:
Would it make sense to build the `BUILDERS` part of this string as we go loop through
```
...
BUILDERS_PUSH_STRING="jenkinsxio/builder-${i} ${VERISON} ${BUILDERS_PUSH_STRING}
...
```
Then we could just add `${BUILDERS_PUSH_STRING}` to the command.